### ISSUE TITLE
Bad HTML export of [TOC] on Windows

### ISSUE THREAD

**Issue by arcr-dev** [rhiokim/haroopad#484]
[TOC]
...

exports to

``` html
<p class="toc" style="undefined"></p>
....
<p></p>
```

when it should be

``` html
<p class="toc" style="undefined">
....
</p>
```

but this would not work either, because chrome doesn't accept other html elements inside p

so it should be

``` html
<div class="toc" style="undefined">
....
</div>
```

and the style should be div.toc instead of p.toc

**Comment by k1ngf15h3r** [comment]
I realized that I have a second dot in front of my toc content. I think there is no dot needed in the toc. Would be nice when it looks like a book toc and if the page number will be displayed at the right side of the headline

**Comment by rhiokim** [comment]
@arcr-dev Thanks idea.
It'll be fixed with v0.13.2
